Resources/Prototypes/_Moffstation/Recipes/Cooking/meal_recipes.yml

```diff
@@ -19,3 +19,25 @@
   solids:
     FoodBreadBun: 1
     ClothingEyesGlassesJamjar: 1
+
+- type: microwaveMealRecipe
+  id: RecipeMothroachDeathsheadBurger
+  name: mothroach burger recipe
+  result: FoodBurgerMothRoachDeathshead
+  secretRecipe: true
+  time: 10
+  group: None # I don't know if this is the correct way to hide it from the guidebook, but it doesn't seem to create errors.
```

Comment:
I think `group` should be the same as the base burger's, and instead you should set `secretRecipe: true`